schema/playground/launcher.schema

```diff
@@ -7,10 +7,11 @@ import "improbable/vector3.schema";
 
 // Command sent by clients to request launching a Launchable.
 type LaunchCommandRequest {
-    int64 entity_to_launch = 1;
+    EntityId entity_to_launch = 1;
     improbable.Vector3f impact_point = 2;
     improbable.Vector3f launch_direction = 3;
     float launch_energy = 4;
+    EntityId player = 5;
```

Comment:
This is only sent from a player entity top the player entity. So this field isn't needed.

Reply:
If this command is now being sent to other entities then this probably needs to be redesigned.

Reply:
Not exactly, it is sent to a Cube entity: it is needed to keep track of the most recent player that shot the cube.